livox_ros_driver/livox_ros_driver/lddc.cpp

```diff
@@ -98,6 +97,24 @@ Lddc::~Lddc() {
   }
 }
 
+void Lddc::SetRosNode(ros::NodeHandle *node) {
+  cur_node_ = node;
+
+  // Fill in covariance matrices

```

Comment:
I don't like his this increases the scope of the `SetRosNode()` function. I think that we should have a new function `lddc::SetImuCovarance()` that this logic could be put into.